site/docs/uk/plugins/session.md

```diff
@@ -581,20 +587,22 @@ bot.start();
 У свою чергу, вам потрібно буде налаштувати кожен фрагмент за допомогою їх власної конфігурації.
 
 ```ts
-bot.use(session({
-  type: "multi",
-  foo: {
-    // це також значення за замовчуванням
-    storage: new MemorySessionStorage(),
-    initial: () => undefined,
-    getSessionKey: (ctx) => ctx.chat?.id.toString(),
-  },
-  bar: {
-    initial: () => ({ prop: 0 }),
-    storage: freeStorage(bot.token),
-  },
-  baz: {},
-}));
+bot.use(
+  session({
+    type: "multi",
+    foo: {
+      // це також типове значення значення
```

Comment:
```suggestion
      // це також типове значення
```

I think it's still clear what we're talking about if we remove one word.

Reply:
I corrected a language mistake and did not notice the duplication of the word. There is no hidden meaning here 😅